src/IconButton/IconButton.spec.js

```diff
@@ -72,4 +77,12 @@ describe('<IconButton />', () => {
       assert.include(wrapper.props().style, hoveredStyle);
     });
   });
+  describe('prop: disabled', () => {
+    it('disables the ripple effect', () => {
+      const wrapper = mountWithContext(
+        <IconButton disabled={true} />
```

Comment:
Could we add another test with `disabled={false}` to makes sure `(wrapper.find(TouchRipple).length === 1`?